helm-charts/verticadb-operator/tests/kind-concurrency_test.yaml

```diff
@@ -8,6 +8,7 @@ tests:
         verticadb: 8
         verticaautoscaler: 2
         eventtrigger: 3
+        verticarestorepointsquery: 1
```

Comment:
It's good that we have a unit test, but we should use a value other than the default. Just to prove that it is working.